tests/test_parameters.py

```diff
@@ -3,34 +3,61 @@
 
 
 class TestParameters(unittest.TestCase):
+    def setUp(self):
```

Comment:
What is the difference between setUp and setUpClass? Is this setUp called before each individual test in the class, or is that redone for each individual test_*?

Reply:
I have never used [setUpClass](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass) before. Anyways, `setUp()` is called immediately before each test method, here -- in the case of `TestParameters` with 5 test -- 5 times.